jsapp/js/components/multiCheckbox.es6

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import autoBind from 'react-autobind';
+import Checkbox from 'js/components/common/checkbox';
+
+/**
+ * @namespace MultiCheckboxItem
+ * @prop {boolean} checked
+ * @prop {boolean} disabled
+ * @prop {string} label
+ */
+
+/**
+ * A MultiCheckbox generic component.
+ *
+ * @prop {MultiCheckboxItem[]} items
+ * @prop {function} onChange - returns whole list whenever any item changes
+ */
+class MultiCheckbox extends React.Component {
+  constructor(props){
+    super(props);
+    autoBind(this);
+  }
+
+  onChange(response) {
```

Comment:
It's nice you've bound the `onChange` to index, but it means it should be `(itemIndex, isChecked)`.